OpenSim/Simulation/Manager/Manager.h

```diff
@@ -173,11 +167,25 @@ class OSIMSIMULATION_API Manager
     { _writeToStorage =  writeToStorage; }
 
     // Integrator
+    enum class Integrator {
+        CPodes,
+        ExplicitEuler,
+        RungeKutta2,
+        RungeKutta3,
+        RungeKuttaFeldberg,
+        RungeKuttaMerson,
+        //SemiExplicitEuler, no error ctrl, requires fixed stepSize arg on construction
+        SemiExplcitEuler2,
+        Verlet
+    };
+    void setIntegrator(Integrator integMethod);
+    void setIntegrator(const std::string& integMethod);
     SimTK::Integrator& getIntegrator() const;
-    /** %Set the integrator. The Manager does *not* take ownership of the
-     * passed-in integrator.
-     */
-    void setIntegrator(SimTK::Integrator&);
```

Comment:
I think it would improve clarity to prefix these methods with "Integrator" (e.g., `setIntegratorAccuracy()`).